nautobot_chatops_ipfabric/worker.py

```diff
@@ -881,24 +910,24 @@ def find_host(dispatcher, filter_key=None, filter_value=None):
                 "Host Inventory",
                 ipfabric_logo(dispatcher),
             ),
-            dispatcher.markdown_block(f"{ipfabric_api.host_url}/inventory/hosts"),
+            dispatcher.markdown_block(f"{ipfabric_api.ui_url}inventory/hosts"),
         ]
     )
 
     dispatcher.send_large_table(
         ["Host IP", "VRF", "Host DNS", "Site", "Edges", "Gateways", "Access Points", "Host MAC", "Vendor", "VLAN"],
         [
             (
-                host.get("ip") or "(empty)",
-                host.get("vrf") or "(empty)",
-                host.get("dnsName") or "(empty)",
-                host.get("siteName") or "(empty)",
-                host.get("edges") or "(empty)",
-                host.get("gateways") or "(empty)",
-                host.get("accessPoints") or "(empty)",
-                host.get("mac") or "(empty)",
-                host.get("vendor") or "(empty)",
-                host.get("vlan") or "(empty)",
+                host.get(IpFabric.ADDRESSING_HOSTS_COLUMNS[0]) or IpFabric.EMPTY,
```

Comment:
(host.get(IpFabric.ADDRESSING_HOSTS_COLUMNS[1], IpFabric.EMPTY) for i in range(len(IpFabric.ADDRESSING_HOSTS_COLUMNS))